S4TF_Tutorial_1.ipynb

```diff
@@ -331,7 +331,7 @@
         "\n",
         "Python has lists built into the language.\n",
         "However, we will use a library called numpy for this.\n",
-        "Numpy gives you lot's of support functions that are useful when doing Machine Learning.\n",
+        "Numpy gives you lots of support functions that are useful when doing Machine Learning.\n",
         "\n",
         "Here, you will also see an import statement. This statement makes the entire numpy package available and we can access those symbols using the abbreviated 'np' syntax."
```

Comment:
Maybe you could phrase this like the following:

"In the following, you will see an import statement that makes the entire NumPy package available to Swift. By assigning this imported library to `np`, we can access NumPy symbols using an abbreviated syntax."